azurerm/internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -103,6 +116,17 @@ func resourceArmKubernetesClusterNodePool() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				//ForceNew: true,
```

Comment:
Do we need this commented?

Reply:
this wants removing - in testing node pools can be migrated